circuits/cpp/src/aztec3/circuits/abis/private_old_tree_roots.hpp

```diff
@@ -0,0 +1,103 @@
+#pragma once
+#include <barretenberg/stdlib/primitives/witness/witness.hpp>
+#include <aztec3/utils/types/native_types.hpp>
+#include <aztec3/utils/types/circuit_types.hpp>
+#include <aztec3/utils/types/convert.hpp>
+
+namespace aztec3::circuits::abis {
+
+using aztec3::utils::types::CircuitTypes;
+using aztec3::utils::types::NativeTypes;
+using plonk::stdlib::witness_t;
+using std::is_same;
+
+template <typename NCT> struct PrivateHistoricTreeRoots {
```

Comment:
I'm already seeing a file called `private_historic_tree_roots` on my local version of `master` which contains this struct. Perhaps this weirdness is as a result of @LHerskind recently renaming things from 'old' to 'historic' in places?

Reply:
Yes, I'll delete the 'old' versions of this and the combined struct.